### PR TITLE
ica: TrySendTx error handling nits

### PR DESCRIPTION
**Pull request by damiannolan**
## Description

Picking a couple of error handling nits from #447 

- TrySendTx (relay.go) channel not found should return for which port id
- Add channel/port id in get next send sequence not found error (createOutgoingPacket - keeper/relay.go)
- ErrUnkownPacketData -> ErrUnknownDataType (errors.go)

partially closes: #447 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/27-interchain-accounts/keeper/relay.go [hunk 1]
```diff
@@ -14,20 +14,20 @@ import (
 // manage helper module access to owner addresses they do not have capabilities for
 func (k Keeper) TrySendTx(ctx sdk.Context, portID string, icaPacketData types.InterchainAccountPacketData) (uint64, error) {
 	// Check for the active channel
-	activeChannelId, found := k.GetActiveChannel(ctx, portID)
+	activeChannelID, found := k.GetActiveChannel(ctx, portID)
```
[the remark this answers is not in the document]
Comment (seantking) [reply modules/apps/27-interchain-accounts/keeper/relay.go:17]: Agreed :+1:

File: modules/apps/27-interchain-accounts/keeper/relay.go [hunk 2]
```diff
@@ -171,7 +171,7 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet) error
 	var data types.InterchainAccountPacketData
 
 	if err := types.ModuleCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
-		return sdkerrors.Wrapf(types.ErrUnknownPacketData, "cannot unmarshal ICS-27 interchain account packet data")
+		return sdkerrors.Wrapf(types.ErrUnknownDataType, "cannot unmarshal ICS-27 interchain account packet data")
```
[the remark this answers is not in the document]
Comment (seantking) [reply modules/apps/27-interchain-accounts/keeper/relay.go:174]: Yes, I think you are correct.

[the remark this answers is not in the document]
Comment (seantking) [reply modules/apps/27-interchain-accounts/keeper/relay.go:174]: I guess we can remove the custom error type as well

**Review by seantking — APPROVED** [review]

File: modules/apps/27-interchain-accounts/keeper/relay.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (colin-axner) [reply modules/apps/27-interchain-accounts/keeper/relay.go:17]: :+1:

File: modules/apps/27-interchain-accounts/keeper/relay.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (colin-axner) [reply modules/apps/27-interchain-accounts/keeper/relay.go:174]: No we should not. The err message is indeterminate (this gets written into failed ack)

**Review by colin-axner — APPROVED** [review]

File: modules/apps/27-interchain-accounts/keeper/relay.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (damiannolan) [reply modules/apps/27-interchain-accounts/keeper/relay.go:174]: Ahh! Yes of course 💯

[the remark this answers is not in the document]
Comment (damiannolan) [reply modules/apps/27-interchain-accounts/keeper/relay.go:175]: Added a comment! 👍

File: modules/apps/27-interchain-accounts/keeper/relay.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (damiannolan) [reply modules/apps/27-interchain-accounts/keeper/relay.go]: Done

**Pull request merged**